apps/core/src/hooks/useTransactionSummary.ts

```diff
@@ -22,6 +23,12 @@ const getSummary = (
     const objectSummary = getObjectChangeSummary(transaction, currentAddress);
     const balanceChangeSummary = getBalanceChangeSummary(transaction);
 
+    const totalChangeCount =
```

Comment:
Do we need a specific number, or should we pass in a variable like `totalChangeThreshold` and return a boolean `isExceedingObjectCount` or something instead. That way we can exit the count early, in the event we might have a lot of objects to count.

Reply:
I'll probably just split up the PR again into adding details and fixing mobile layout, and addressing some of these larger issues